test/minitest_helper.rb

```diff
@@ -39,6 +39,12 @@ class HistoryWorker
   end
 end
 
+def travel_to(time)
```

Comment:
I think it's important to isolate the `Time.stub` behavior here in order to scale it in the future.